backend/src/main/java/com/github/esgoet/backend/services/BookService.java

```diff
@@ -28,4 +29,15 @@ public Book getBook(String id) {
         return bookRepository.findById(id)
                 .orElseThrow(() -> new BookNotFoundException("No book found with id: " + id));
     }
+
+    public Book saveNewABook(NewBookDto newBookDto) {
```

Comment:
again, would use saveBook or at least saveANewBook
```suggestion
    public Book saveBook(NewBookDto newBookDto) {
```